### PR TITLE
Use the new unified web container image for user-guide

### PR DESCRIPTION
**Pull request by cwilkers**
Should fix https://github.com/kubevirt/user-guide/issues/623

Signed-off-by: cwilkers <cwilkers@redhat.com>

### CODE REVIEW

**Comment by cwilkers** [comment]
CC @dhiller

File: github/ci/prow-deploy/files/jobs/kubevirt/user-guide/user-guide-postsubmits.yaml [hunk 1]
```diff
@@ -17,13 +17,13 @@ postsubmits:
       securityContext:
         runAsUser: 0
       containers:
-      - image: docker.io/library/python:3.7
+      - image: kubevirt-kubevirt.github.io:v20220825-9d7149f
```
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/user-guide/user-guide-postsubmits.yaml:20]:
```suggestion
      - image: quay.io/kubevirtci/kubevirt-kubevirt.github.io:v20220825-9d7149f
```

**Review by dhiller — CHANGES_REQUESTED** [review]
@cwilkers thank you for this!

I think you need to correct one image reference to point to quay, other than that looks good to me!

**Review by dhiller — APPROVED** [review]
/approve

**Pull request merged**